### DISCUSSION TITLE
Google PlatStore Update

### DISCUSSION THREAD

**Discussion by Bhardwaj10Ankit** [LawnchairLauncher/lawnchair#5062] [General]
it's been 3 months from the last [Lawncair (Early Access)](https://play.google.com/store/apps/details?id=app.lawnchair.play) update on Google Play Store, which was also the first launch of 14dev. Why not any ore updates on PlayyStore???

**Comment by validcube** [comment]
Lawnchair on Google Play Store will be updated once a new "early release" (like alpha, beta) gets released. Unlike on GitHub where we offered Nightly (newer, unstable and slow version of Lawnchair) along side with early releases.

**Reply by eduhoribe** [reply]
Now that we have a new stable-_ish_ release, is the update on the Play Store underway? I know that review process from Google can take quite a while.

**Comment by Akash-Sriram** [comment]
It's been years why actually no stable release in GitHub itself and playstore updates are way outdated compare to GitHub

**Comment by SuperDragonXD** [comment]
Hello! I know that this reply is a bit late, but as seen [in our FAQ](https://lawnchair.app/faq/#when-is-the-next-update), updates depend on volunteer availability (we had a Lawnchair 15 Beta 2 update a few days ago). We also won't make a "true" stable release until almost all the features of Lawnchair v2 are in the newer versions while having lesser bugs.
